src/gennylib/include/gennylib/context.hpp

```diff
@@ -518,11 +521,26 @@ class PhaseContext final : public v1::ConfigNode {
         return _actor->workload();
     }
 
+    /**
+     * Convenience method for creating a metrics::Operation that's unique for this phase and thread.
```

Comment:
may be worth noting that if the `Phase:` block has a `Name:` param that will be used else it will inherit?